src/main/java/com/yetanalytics/xapi/model/LangMap.java

```diff
@@ -9,32 +9,62 @@
 import com.yetanalytics.xapi.model.deserializers.LangMapDeserializer;
 import com.yetanalytics.xapi.model.serializers.LangMapSerializer;
 
+/**
+ * Java wrapper object for the xAPI Language Map object.
+ * The Language Map is a dictionary where the keys are 
+ * <a href="http://tools.ietf.org/html/rfc5646">RFC 5646 Language Tags</a> and
+ * the value is a String in the language specified by the tag.
+ */
```

Comment:
You could potentially link to the Language Map definition in the xAPI spec: https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#42-language-maps

In particular, it would be useful to add that to the first sentence since the API page only shows the first sentence of each class' Javadoc string.

Reply:
59646edfc8cb2d3de629b961452f5aa15c75c4ac